### ISSUE TITLE
git-plus status returns "No matches found"

### ISSUE THREAD

**Issue by erelsgl** [akonwi/git-plus#501]
When I click Ctrl+Shift+H and then select Status, I get a grey message saying "No matches found".

When I to "git status" in my project root, I get:

```
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean

```

**Comment by mlgann** [comment]
Im having the same issue.

**Comment by akonwi** [comment]
This was fixed with #448
